api/v1/drivemgrsvc.proto

```diff
@@ -13,6 +13,16 @@ message DrivesResponse {
     repeated Drive disks = 1;
 }
 
+message DriveLocateRequest {
+    string driveSerialNumber = 1;
+    string action = 2;
```

Comment:
Using `string` here seems to be redundant. Suggest to have `integer` instead:

> 0 - start
> 1 - stop
> 2 - status

Reply:
done